dockerfiles/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
+services:
+  hpp-demo:
+    init: true
+    image: hpp-demo
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix
+    environment:
+      - DISPLAY=${DISPLAY}
+      - QT_X11_NO_MITSHM="1"
```

Comment:
I don't think you would need a graphical stack for hppcorbaserver

Reply:
Have you configured `xhost` to allow relevant connections ?

Reply:
That's totally right, thanks :)